free-1.md

```diff
@@ -57,12 +136,48 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+I think around 50 percent of the time would be spent on desigining the language.
+I feel like it will be fairly simple to define terms of the lanugage, especially
+because coding languages for visual art already exist and I would be able to model
+aspects of my language after those. However, I think I'll need to make important 
+design decisions on how the the code can be organized from the user end (i.e. 
+creating classes or separate files). I also think I'll need to decide how simple
+the language will be, either by limiting the vocabulary or by limiting the
+parameters for different objects.
```

Comment:
One thing you might consider is whether you would want one method to take a different number of parameters. For example, you could have default parameters if a user doesn't specify them. 
Under a design decision with a variable # of parameters, assume that the default fill color was black, the default outline color was white, and the default radius was 1 unit.
If you wanted to create a circle with a black fill, white outline, and 1-unit radius, you could just type: "circle()"
However, if you wanted to create a circle with a red fill, purple outline, and 1-unit radius, you could just type: "circle(red, purple)"